### PR TITLE
added styling and function to the carousel

### PR DESCRIPTION
**Pull request by FelixHultman**
This is a Moviecarousel/slider where the user look through the movies playing now and click on them to get to the moviepage.

The border around the posters maybe looks a bit iffy but it as actually worse without it in my opinion so i made that decision.

the rest is pretty straightforward i think.

https://github.com/JrSchoolAccount/Kino-site-nextjs/assets/145000258/6cb53f14-6837-4bb5-98fd-1c1e4bb3d1d2

### CODE REVIEW

**Comment by MasakiShiraishi** [comment]
Hi.
My opinion is that both sides of the sliders are a little bit small, making them difficult to see. It would be better if they were a bit bigger. However, I'm not entirely certain about that. Nevertheless, the slider function seems to be working fine, and I think it's great.

**Review by rl-86 — APPROVED** [review]
Looks good! 
Maybe the headline "Akutella Filmer" might be a little bit big and it doesn't change size with smaller screens, otherwise, good job! 👍

**Comment by FelixHultman** [comment]
> Looks good! Maybe the headline "Akutella Filmer" might be a little bit big and it doesn't change size with smaller screens, otherwise, good job! 👍

I made i a bit smaller now

**Comment by FelixHultman** [comment]
> Hi. My opinion is that both sides of the sliders are a little bit small, making them difficult to see. It would be better if they were a bit bigger. However, I'm not entirely certain about that. Nevertheless, the slider function seems to be working fine, and I think it's great.

I think it would be better if they were a bit bigger as you, if i go back to some more work on it later i will implement it then but i think this will work for now

File: app/components/movieCarousel.tsx [hunk 1]
```diff
@@ -1,58 +1,97 @@
...
 import { carouselSettings } from '../lib/definitions';
+import Image from 'next/image';
+import { Movie } from '../lib/definitions';
+import { useRouter } from 'next/navigation';
 
 const MovieCarousel = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const router = useRouter();
+
+  const handleClick = (movieId: any) => {
```
Comment (NelsonTheBrave) [inline app/components/movieCarousel.tsx:16]: There is actually a specific type for mongoose id:s that you could use (mongoose.Types.ObjectId). Just to let you know :)

**Review by NelsonTheBrave — APPROVED** [review]
Nice. now the page starts to actually look a bit decent :D
I agree with previous comment about the size of arrow button but it's not essential. 
I noticed a general issue with that the whole page is scrollable in x-direction, which makes your carousel end up a bit outside the screen on smaller widths (mostly just the arrow). But this is not your code, it must have happened earlier

**Pull request merged**